components/ProfileDropdown.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import { View, TouchableOpacity, Image, Text, StyleSheet, useColorScheme } from 'react-native';
+import Popover from 'react-native-popover-view';
+import { useRouter } from 'expo-router';
+
+const ProfileDropdown = () => {
+  const colorScheme = useColorScheme();
+
+  const containerStyle = colorScheme === 'dark' ? styles.darkContainer : styles.lightContainer;
+  const textStyle = colorScheme === 'dark' ? styles.darkText : styles.lightText;
+  const borderColor = colorScheme === 'dark' ? styles.darkBorder : styles.lightBorder;
+
+  const [isVisible, setIsVisible] = useState(false);
+  const router = useRouter();
+
+  const togglePopover = () => {
+    setIsVisible(!isVisible);
+  };
+
+  const handleNavigate = (route:any) => {
+    setIsVisible(false);
+    router.push(route);
+  };
+
+  return (
+    <View style={styles.container}>
+      <Popover
+        isVisible={isVisible}
+        onRequestClose={togglePopover}
+        from={
+          <TouchableOpacity onPress={togglePopover}>
+            <Image source={require('@/assets/images/profile.png')} style={styles.profileImage} />
+          </TouchableOpacity>
+        }
+      >
+        <View style={[styles.dropdownMenu, containerStyle, borderColor]}>
+          <TouchableOpacity
+            style={styles.menuItem}
+            className='border-b border-gray-300'
+            onPress={() => handleNavigate('/dashboard/trainee/Profile')}
+          >
+            <Text style={textStyle}>Profile</Text>
+          </TouchableOpacity>
+          <TouchableOpacity
+            style={styles.menuItem}
+            onPress={() => handleNavigate('/dashboard/trainee/preference')}
+          >
+            <Text style={textStyle}>Settings</Text>
+          </TouchableOpacity>
+        </View>
+      </Popover>
+    </View>
+  );
+};
+
+// Define styles
+const styles = StyleSheet.create({
```

Comment:
We moved from native styles to tailwind, please use tailwind in your styles

Reply:
Resolved